### DISCUSSION TITLE
Record plugin - how to render the waveform in a "left-to-right" timeline manner (instead of from the right side)

### DISCUSSION THREAD

**Discussion by tyler-g** [katspaugh/wavesurfer.js#3830] [Q&A]
Looking through the Record plugin options, I see there are the options to render the audio as waveform, and also to set the window length in seconds of the rendering area, 
but I cannot figure out how to render the audio left-to-right, as you would normally see it in a DAW.   Is this possible?

**Comment by katspaugh** [comment]
It's not possible out of the box but PRs are welcome.
